gammapy/irf/edisp/map.py

```diff
@@ -538,3 +538,34 @@ def peek(self, figsize=(15, 5)):
 
         """
         self.get_edisp_kernel().peek(figsize)
+
+    @classmethod
+    def read(cls, filename, format="gadf", hdu=None, checksum=False):
+        """Read an EDispKernelMap from file and create corresponding object.
```

Comment:
Here I would explain that this does not only read a map but create one assuming spatial uniformity when format is drm. 
Ideally this would deserve a different classmethod, but I suppose this is fine like this.

Reply:
added

Reply:
@registerrier The good thing is, that the this information is still explicit in the geom of the resulting map. So no meta info is lost and creating missing one on I/O is fine.